addons/web/static/src/webclient/actions/reports/bootstrap_overridden_report.scss

```diff
@@ -2,3 +2,17 @@ $link-decoration: none !default;
 
 // remove Emoji fonts
 $font-family-sans-serif: o-add-unicode-support-font(("Lucida Grande", Helvetica, Verdana, Arial, sans-serif), 1);
+
+// Boostrap 5 introduces variable paddings for container which wkhtmltopdf doesn't seem to process, so we restore Boostrap 4's paddings for PDFs
+.container {
+    padding-right: 15px;
+    padding-left: 15px;
```

Comment:
use Bootstrap SCSS variable instead of magic number

Reply:
No there is a difference between `SCSS variables` [1] and `CSS Custom properties` [2].
In this case i'm referencing the SCSS variable `$gutter` [3].
This variable is the same as `$grid-gutter-width`.

So for the standard Bootstrap 4, `15px` is the right value.
But in Odoo we customize Bootstrap so we have file called `bootstrap_overridden.scss` that override the default value of Bootstrap 5, we only have the override this part for the backend where the value is `16px` but in the report the value is still the default `0.75rem`.

```scss
/* Code from Bootstrap 4 */
$grid-gutter-width: 30px !default;

@mixin make-container($gutter: $grid-gutter-width) {
  width: 100%;
  padding-right: $gutter / 2;
  padding-left: $gutter / 2;
  margin-right: auto;
  margin-left: auto;
}

@if $enable-grid-classes {
  .container {
    @include make-container();
    @include make-container-max-widths();
  }
}

/* Code from Boostrap 5 */
$grid-gutter-width: 1.5rem !default;

$container-padding-x: $grid-gutter-width * .5 !default;

@mixin make-container($gutter: $container-padding-x) {
  width: 100%;
  padding-right: var(--#{$variable-prefix}gutter-x, #{$gutter});
  padding-left: var(--#{$variable-prefix}gutter-x, #{$gutter});
  margin-right: auto;
  margin-left: auto;
}

@if $enable-grid-classes {
  .container, .container-fluid {
    @include make-container();
  }
}

/* Odoo override for backend (webclient) */
$o-horizontal-padding: 16px !default;
$grid-gutter-width: $o-horizontal-padding * 2 !default;
```

```suggestion
    padding-right: $container-padding-x;
    padding-left: $container-padding-x;
```
and add the SCSS variable `$container-padding-x` at the top of the file (`bootstrap_overridden_report.scss`):
```scss
$container-padding-x: 15px; /* Like in BS4 */
/* or */
$container-padding-x: 16px; /* Like in BS5 */
```

These lines (`$container-padding-x: ...`) help to grep/find the origin of the value if need in the future and also get rid of the usage of `CSS Custom properties` (`var(--...)`).

[1] https://sass-lang.com/documentation/variables
[2] https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Variables
[3] https://github.com/twbs/bootstrap/blob/e5643aaa89eb67327a5b4abe7db976f0ea276b70/scss/mixins/_grid.scss#L7

Reply:
Thanks a lot for the thorough explanation, makes way more sense for me now. I did as you suggested.